OpenSim/Simulation/Model/Model.h

```diff
@@ -925,17 +925,17 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Model, ModelComponent);
     /**
      * Print some basic information about the model.
      *
-     * @param aOStream Output stream.
+     * @param aOStream Output stream (e.g., std::cout).
```

Comment:
+1.

I also wish the stream had a default argument, so that these functions could be used in MATLAB/python. But that is unrelated to this PR; no need to do anything.